sdk/identity/identity/src/credentials/authorityValidationOptions.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Provides options to configure how the Identity library 
+ * does authority validation during authentication requests
+ * to Azure Active Directory.
+ */
+export interface AuthorityValidationOptions {
+    knownAuthorities?: Array<string>;
```

Comment:
Spacing seems to be 4 instead of 2 spaces.

Reply:
updated